packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -258,7 +276,14 @@ export const LocationDetailViewTable = (): JSX.Element => {
         } else if (column.key === ('download' as keyof LocationItem)) {
           return <DownloadControl fileKey={row.key} />;
         } else {
-          return row[column.key];
+          return (
+            <>
+              {column.key === 'key' && row.type === 'FILE' ? (
+                <TableDataIcon variant="file" />
```

Comment:
Can we just use `IconElement` and pass the `classname`/`variant` directly here?

Reply:
Yeah that works, updated in the latest.